<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QHBoxLayout" >
    <property name="spacing" >
     <number>0</number>
    </property>
    <property name="leftMargin" >
     <number>0</number>
    </property>
    <property name="topMargin" >
     <number>0</number>
    </property>
    <property name="rightMargin" >
     <number>0</number>
    </property>
    <property name="bottomMargin" >
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="dataTab" >
      <property name="currentIndex" >
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab" >
       <attribute name="title" >
        <string>Data</string>
       </attribute>
       <layout class="QVBoxLayout" >
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QTableWidget" name="tableWidget" />
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="kmeansTab" >
       <attribute name="title" >
        <string>K-Means</string>
       </attribute>
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>0</number>
        </property>
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QwtPlot" native="1" name="kmeansPlot" />
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="claraTab" >
       <attribute name="title" >
        <string>Clara</string>
       </attribute>
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>0</number>
        </property>
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QwtPlot" native="1" name="claraPlot" />
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QDockWidget" name="dockWidget" >
      <property name="windowTitle" >
       <string>Results</string>
      </property>
      <widget class="QWidget" name="dockWidgetContents_2" >
       <layout class="QVBoxLayout" >
        <property name="rightMargin" >
         <number>9</number>
        </property>
        <property name="bottomMargin" >
         <number>4</number>
        </property>
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>Number of centroids</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="centroidsNumber" >
            <property name="minimum" >
             <number>1</number>
            </property>
            <property name="maximum" >
             <number>10</number>
            </property>
            <property name="singleStep" >
             <number>1</number>
            </property>
            <property name="value" >
             <number>4</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton" >
            <property name="text" >
             <string>Process</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox" >
          <property name="title" >
           <string>Results</string>
          </property>
          <layout class="QVBoxLayout" >
           <item>
            <widget class="QTreeView" name="treeView" />
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton" >
             <property name="text" >
              <string>K-Means</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_2" >
             <property name="text" >
              <string>Clara</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>610</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionClearDataSet" />
   <addaction name="actionLoad" />
   <addaction name="actionRandom2D" />
   <addaction name="actionRandom3D" />
   <addaction name="actionSwitchView" />
   <addaction name="actionProcess" />
   <addaction name="separator" />
   <addaction name="actionQuit" />
  </widget>
  <action name="actionLoad" >
   <property name="text" >
    <string>Load</string>
   </property>
   <property name="shortcut" >
    <string>F3</string>
   </property>
  </action>
  <action name="actionRandom2D" >
   <property name="text" >
    <string>Random &amp;2D</string>
   </property>
  </action>
  <action name="actionRandom3D" >
   <property name="text" >
    <string>Random &amp;3D</string>
   </property>
  </action>
  <action name="actionQuit" >
   <property name="text" >
    <string>Quit</string>
   </property>
  </action>
  <action name="actionProcess" >
   <property name="text" >
    <string>Process</string>
   </property>
  </action>
  <action name="actionClearDataSet" >
   <property name="text" >
    <string>Clear</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSwitchView" >
   <property name="text" >
    <string>SwitchView</string>
   </property>
   <property name="shortcut" >
    <string>F12</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QWidget</extends>
   <header location="global" >qwt_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
